package.json

```diff
@@ -53,7 +53,9 @@
     "typescript": "4.2.3"
   },
   "peerDependencies": {
-    "@vue/compiler-sfc": "3.x",
     "prettier": "2.x"
+  },
+  "optionalDependencies": {
+    "@vue/compiler-sfc": "3.x"
```

Comment:
Optional dependencies are always attempted to be installed by the package manager, but the installation will not fail if it does not complete.  This is intended to be used for packages that can only run in one architecture, for instance. 

The correct approach here is to use `peerDependenciesMeta` like this: https://github.com/IanVS/prettier-plugin-sort-imports/pull/45/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R72-R75

Reply:
Good call; you're definitely right. Just updated the PR w/ this change.